### PR TITLE
Created candidate view component and made candidates clickable.

### PR DESCRIPTION
**Pull request by carlosm4247**
Description:
Created the basic candidate view component and added placeholder content. Additionally, I made candidates in the ResultsBox clickable and added alternating styling when clicked. This will be used in future PRs to display candidate view components for each of the user's clicked candidates in the result box, and they will be removed upon clicking the same candidate again.

Milestones:
- User can compare candidates from federal races
- User can follow candidates/races and get news on them in a feed

Test Details:

https://github.com/carlosm4247/capstone-project/assets/126618515/07da57a3-ad13-4dbf-98b8-102c9bd1216d

### CODE REVIEW

File: capstone-ui/src/Components/CandidateView/CandidateView.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
+
+export default function CandidateView({ candidateName }) {
```
Comment (nick-r-117) [inline capstone-ui/src/Components/CandidateView/CandidateView.jsx:3]: I don't see this used.

File: capstone-ui/src/Components/ResultsBox/ResultsBox.jsx [hunk 2]
```diff
@@ -59,6 +59,19 @@ export default function ResultsBox( { locationLevel, countyFIPS } ) {
 
     candidates.sort((a, b) => b.voteCount - a.voteCount);
 
+    const [selectedCandidates, setSelectedCandidates] = useState([])
+
+    function handleClick(key) {
+        setSelectedCandidates((selected) => {
```
Comment (nick-r-117) [inline capstone-ui/src/Components/ResultsBox/ResultsBox.jsx:65]: Is `selected` the current state value?

File: capstone-ui/src/Components/ResultsBox/ResultsBox.jsx [hunk 3]
```diff
@@ -59,6 +59,19 @@ export default function ResultsBox( { locationLevel, countyFIPS } ) {
...
+            if (selected.includes(key)) {
+                return selected.filter((candKey) => candKey != key);
+            }
+            else {
+                return selected.concat(key);
```
Comment (nick-r-117) [inline capstone-ui/src/Components/ResultsBox/ResultsBox.jsx:70]: Does this take into account max candidates selected as 2?

**Review by nick-r-117 — APPROVED** [review]

File: capstone-ui/src/Components/ResultsBox/ResultsBox.jsx (hunk 2 above)
Comment (carlosm4247) [reply capstone-ui/src/Components/ResultsBox/ResultsBox.jsx:65]: Yes

File: capstone-ui/src/Components/ResultsBox/ResultsBox.jsx (hunk 3 above)
Comment (carlosm4247) [reply capstone-ui/src/Components/ResultsBox/ResultsBox.jsx:70]: Not yet, that's added later on. For this moment, it just allows everything to be selected.

File: capstone-ui/src/Components/CandidateView/CandidateView.jsx (hunk 1 above)
Comment (carlosm4247) [reply capstone-ui/src/Components/CandidateView/CandidateView.jsx:3]: Oh, I thought I needed to import React for every component. I just looked it up and learned that in the most current version, you don't have to unless you're using one of the features. I'll remove it from all of the components that don't need it.

**Pull request merged**